server.js

```diff
@@ -77,6 +78,9 @@ function initialiseGlobalMiddleware (app) {
   app.use(compression())
 
   app.disable('x-powered-by')
+
+  logger.info('Heap statistics: ' + v8.getHeapStatistics())
```

Comment:
This code will only execute once when the app starts up. Is that the intention?

Reply:
i think for now yes. @heathd was this what you were suggesting this morning, that we log memory stats when the app starts up? `getHeapStatistics` will give us the total heap size (see https://nodejs.org/api/v8.html#v8_v8_getheapstatistics)